src/granite/type.cr

```diff
@@ -62,24 +48,4 @@ module Granite::Type
   def from_rs(result : DB::ResultSet, t : Time?.class) : Time?
     result.read(Time?).try &.in(Granite.settings.default_timezone)
   end
-
-  {% for type, method in NUMERIC_TYPES %}
```

Comment:
Will this logic be moved to a `from_params` method?  Or are you thinking this will be removed from Granite?

Reply:
It depends on the approach we take with option 1. If we support hash it would just work with `#new`. If we don't want to support hashes I was thinking of another overload that accepts a `HTTP::Params` object.  However, I haven't messed with that at all yet.